cmd/restic/cmd_snapshots.go

```diff
@@ -145,7 +145,7 @@ func (cmd CmdSnapshots) Execute(args []string) error {
 		tab.Rows = append(tab.Rows, []interface{}{sn.ID()[:plen/2], sn.Time.Format(TimeFormat), sn.Hostname, sn.Paths[0]})
 
 		if len(sn.Paths) > 1 {
-			for _, path := range sn.Paths {
+			for _, path := range sn.Paths[1:] {
```

Comment:
The first line for each snapshot contains all the other fields (ID, Date etc), the following lines are just added if there is more than one dir. So this indeed fixes the root cause.